src/components/Layout/Footer.tsx

```diff
@@ -40,6 +58,20 @@ export const Footer = () => {
         <EmailIcon link={EMAIL_URL} />
         <DiscordIcon link={DISCORD_URL} />
       </SocialContainer>
+      <CustomDivider sx={{ mx: '25%', marginTop: 2 }} />
+      <LegalContainer>
+        <span>©{new Date().getFullYear()} The Badge</span>
```

Comment:
```suggestion
        <span>The Badge © Copyright {new Date().getFullYear()}. All rights reserved.</span>
```
"The Badge © Copyright 2022. All rights reserved. "